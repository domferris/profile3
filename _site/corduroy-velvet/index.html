<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>
    Dominic Ferris - Corduroy Velvet

  </title>

  <!-- Fontawesome Stylesheet -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">

  <!-- Styles bundle -->
  <link rel="stylesheet" href="/assets/main.bundle.css">

  <!-- Locomotive Scroll polyfills -->
  <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.6.0/polyfill.min.js"
    crossorigin="anonymous"></script>
  <script nomodule
    src="https://polyfill.io/v3/polyfill.min.js?features=Object.assign%2CElement.prototype.append%2CNodeList.prototype.forEach%2CCustomEvent%2Csmoothscroll"
    crossorigin="anonymous"></script>

  <!-- JavaScript bundle -->
  <script src="/assets/main.bundle.js" defer></script>
</head>
<body class="bg-project text-true-gray-800" data-scroll-container>
  
<section class="max-w-md min-h-screen p-4 mx-auto sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16" data-scroll-section>
  <h2 class="text-4xl md:mb-2 xl:mb-5 md:text-6xl lg:text-7xl xl:text-8xl">
    Corduroy Velvet
  </h2>

  <h4 class="text-lg mt-1.5 md:mt-3.5 mb-3 md:mb-4 xl:mb-5 whitespace-nowrap md:text-2xl lg:text-3xl xl:text-4xl">
    Short film
  </h4>
  
  <div class="mb-3 aspect-w-16 aspect-h-9 md:mb-6">
    <iframe class="w-full h-full" title="vimeo-player" src="https://player.vimeo.com/video/304107088?color=D4D4D4&title=0&byline=0&portrait=0" frameborder="0" allowfullscreen></iframe>
  </div>
  
  <div class="space-y-4 md-space-y-8 lg:text-lg md:mb-7">
    <p>
      Corduroy Velvet is a short music film featuring narrative elements and live performances by Emmitt James, the artist behind the EP that shares the name of this short film.
    </p>

    <p>
      The film stars Emmitt James and Nyzeire McGee. I was the sole creator of everything not relating to audio, which was championed by musician and audio engineer Brendan Lynch-Salamon.
    </p>
  </div>

  <div class="grid grid-cols-1 gap-4 my-4 md:grid-cols-2 md:my-7">
    <img src="../../img/projects/cv-01.png" alt="Television with Nyzeire McGee laughing">
    <img src="../../img/projects/cv-02.png" alt="Emmitt James outside performing live">
    <img class="md:col-span-2" src="../../img/projects/cv-03.png" alt="Over the shoulder shot of Emmitt James talking to Nyzeire McGee">
  </div>
  
  <div class="flex flex-col mt-10 md:mt-12 lg:mt-16">
    <div class="flex">    
      <h2 class="pr-4 ml-auto text-4xl md:text-6xl lg:text-7xl">
        Related
      </h2>

      <span class="w-full my-auto md:h-1.5 h-1 lg:h-2 bg-true-gray-800"></span>
    </div>

    <div class="flex flex-col my-4 md:space-y-1 md:my-5 lg:space-y-2 lg:my-6">
      <a class="items-end inline mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/directing">
        Directing<span class="hidden text-base lg:text-lg sm:inline"> Video reel</span>
      </a>

      <a class="items-end inline mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/golden">
        Golden<span class="hidden text-base lg:text-lg sm:inline"> Music video</span>
      </a>
    </div>
    
    <a class="inline-block mr-auto transition duration-300 md:text-xl lg:text-2xl hover:opacity-60" href="/#projects">
      &lt; View other projects
    </a>
  </div>
</section>
</body>
</html>
