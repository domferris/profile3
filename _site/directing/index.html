<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>
    Dominic Ferris - Directing

  </title>

  <!-- Fontawesome Stylesheet -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">

  <!-- Styles bundle -->
  <link rel="stylesheet" href="/assets/main.bundle.css">

  <!-- Locomotive Scroll polyfills -->
  <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.6.0/polyfill.min.js"
    crossorigin="anonymous"></script>
  <script nomodule
    src="https://polyfill.io/v3/polyfill.min.js?features=Object.assign%2CElement.prototype.append%2CNodeList.prototype.forEach%2CCustomEvent%2Csmoothscroll"
    crossorigin="anonymous"></script>

  <!-- JavaScript bundle -->
  <script src="/assets/main.bundle.js" defer></script>
</head>
<body class="text-true-gray-800" data-scroll-container>
  
<div class="bg-true-gray-300" data-scroll-section>
  <section class="max-w-md min-h-screen p-4 mx-auto sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16">
    <h2 class="text-4xl md:mb-2 xl:mb-5 md:text-6xl lg:text-7xl xl:text-8xl">
      Directing
    </h2>
  
    <h4 class="text-lg mt-1.5 md:mt-3.5 mb-3 md:mb-4 xl:mb-5 whitespace-nowrap md:text-2xl lg:text-3xl xl:text-4xl">
      Video reel
    </h4>
    
    <div class="mb-3 aspect-w-16 aspect-h-9 md:mb-6">
      <iframe class="w-full" title="vimeo-player" src="https://player.vimeo.com/video/253403820?color=D4D4D4&title=0&byline=0&portrait=0" frameborder="0" allowfullscreen></iframe>
    </div>

    <div class="space-y-4 md-space-y-8 lg:text-lg md:mb-7">
      <p>
        Before pivoting into technology, I was a freelance director for seven years working in Los Angeles’ entertainment industry. This is a video reel containing select projects throughout my directing career.
      </p>
  
      <p>
        I worked primarily in the independent music scene documenting artists throughout their creative process. My directing journey started in beauty content, working with brands such as CoverGirl, Teen Vogue, and Seventeen Magazine, and transitioned into the music world after documenting the finishing process of Jesse Boykins III’s album Bartholomew for LANDR.
      </p>
    </div>

    <img class="mt-4 md:my-7" src="../../img/projects/directing-me.png" alt="End credits from one of Dominic's films">
    
    <div class="flex flex-col mt-10 md:mt-12 lg:mt-16">
      <div class="flex">    
        <h2 class="pr-4 ml-auto text-4xl md:text-6xl lg:text-7xl">
          Related
        </h2>

        <span class="w-full my-auto md:h-1.5 h-1 lg:h-2 bg-true-gray-800"></span>
      </div>
  
      <div class="flex flex-col my-4 md:space-y-1 md:my-5 lg:space-y-2 lg:my-6">
        <a class="items-end inline mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/corduroy-velvet">
          Corduroy Velvet<span class="hidden text-base lg:text-lg sm:inline"> Short film</span>
        </a>

        <a class="items-end inline mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/golden">
          Golden<span class="hidden text-base lg:text-lg sm:inline"> Music video</span>
        </a>
      </div>
      
      <a class="inline-block mr-auto transition duration-300 md:text-xl lg:text-2xl hover:opacity-60" href="/#projects">
        &lt; View other projects
      </a>
    </div>
  </section>
</div>
</body>
</html>
