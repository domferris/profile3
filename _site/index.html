<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>
    Dominic Ferris - Dev & Design

  </title>
  
  <link rel="stylesheet" href="/assets/main.bundle.css">
  <script src="/assets/main.bundle.js" defer></script>
</head>
<body class="text-true-gray-800" data-scroll-container>
  
<section class="bg-soft-brown" data-scroll-section>
  <div class="flex flex-col max-w-md min-h-screen p-4 mx-auto sm:max-w-xl lg:max-w-7xl lg:p-16 md:p-8 md:max-w-4xl">
    <div class="grid grid-cols-2">
      <h1 class="text-5xl md:text-8xl xl:text-9xl">Dom<br>Ferris</h1>
      <div class="w-full md:h-1.5 h-1 mt-auto mb-4 xl:h-2.5 lg:h-2 md:mb-8 xl:mb-11 bg-true-gray-800"></div>
    </div>
    <div class="flex flex-col justify-center flex-grow space-y-2">
      <h2 class="text-4xl md:text-6xl xl:text-7xl opacity-40 strikethrough" data-scroll data-scroll-speed="1.3">
        Director and<br>Photographer
      </h2>
      <h2 class="text-4xl md:text-6xl xl:text-7xl" data-scroll data-scroll-speed="1.5">
        Developer and<br>Designer
      </h2>
    </div>
  </div>
</section>
<section class="bg-soft-maroon" data-scroll-section>
  <div class="flex flex-col max-w-md min-h-screen p-4 mx-auto sm:max-w-xl lg:max-w-7xl lg:p-16 md:p-8 md:max-w-4xl">
    <h2 class="text-4xl md:text-6xl lg:text-7xl xl:text-8xl">  
      <span class="flex justify-between"><span>Why</span><span>you</span><span>may</span></span>
      
      <span class="flex justify-between"><span>want</span><span>to</span><span>work</span></span>
      
      <span class="flex justify-between"><span>with</span><span>him</span><span>:</span></span>
    </h2>
    <div class="flex flex-col justify-center flex-grow mt-4 space-y-8 md:space-y-10 xl:px-32 md:px-12 sm:px-10 lg:px-24" data-scroll data-scroll-speed="0.75">
      <h3 class="ml-8 text-4xl xl:ml-36 sm:ml-12 md:ml-16 lg:ml-24 md:text-6xl xl:text-7xl">
        <span class="bg-soft-beige">Curious</span>
      </h3>
      <h3 class="ml-auto text-4xl md:text-6xl xl:text-7xl">
        <span class="bg-medium-maroon">Meticulous</span>
      </h3>
      <h3 class="text-4xl leading-tight md:leading-snug xl:leading-normal md:text-6xl xl:text-7xl">
        Outside<br><span class="bg-soft-gray">of the box</span>
      </h3>
      <h3 class="mx-auto text-4xl md:text-6xl xl:text-7xl">
        <span class="bg-medium-olive">Quietly loud</span>
      </h3>
    </div>  
  </div>
</section>
<section class="bg-soft-olive" data-scroll-section>
  <div class="flex flex-col max-w-md min-h-screen p-4 mx-auto sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16">
    <h2 class="text-4xl md:text-6xl lg:text-7xl xl:text-8xl">
      Other reasons:
    </h2>
    <div class="flex flex-col justify-center flex-grow space-y-2" data-scroll data-scroll-speed="0.6">
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="/projects/reele">
        Reele<span class="hidden text-base lg:text-lg sm:inline"> Web application</span>
      </a>
  
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="https://spicygreenbook.org" target="_blank">
        <span>S<span class="hidden sm:inline">picy </span>G<span class="hidden sm:inline">reen </span>B<span class="hidden sm:inline">ook</span></span><span class="hidden text-base lg:text-lg sm:inline"> Web application</span>
      </a>
        
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="/projects/marches">
        Marches<span class="hidden text-base lg:text-lg sm:inline"> Photo series</span>
      </a>

      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="https://vimeo.com/253403820" target="_blank">
        Directing<span class="hidden text-base lg:text-lg sm:inline"> Video reel</span>
      </a>
  
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="/projects/portraiture">
        Portraiture<span class="hidden text-base lg:text-lg sm:inline"> Photo series</span>
      </a>
  
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="https://vimeo.com/304107088" target="_blank">
        Corduroy Velvet<span class="hidden text-base lg:text-lg sm:inline"> Short film</span>
      </a>
  
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="https://vimeo.com/287382648" target="_blank">
        Power Through<span class="hidden text-base lg:text-lg sm:inline"> Spec commercial</span>
      </a>
  
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="https://www.youtube.com/watch?v=eA9uWxf6OcQ" target="_blank">
        Golden<span class="hidden text-base lg:text-lg sm:inline"> Music video</span>
      </a>
  
      <a class="items-end inline mr-auto text-4xl transition duration-300 sm:mr-0 sm:flex sm:justify-between lg:text-5xl hover:opacity-60" href="/projects/post-supervision">
        Post Supervision<span class="hidden text-base lg:text-lg sm:inline"> Ad collection</span>
      </a>
    </div>
  </div>
</section>
<section class="bg-yellow-gray" data-scroll-section>
  <div class="flex flex-col max-w-md min-h-screen p-4 mx-auto sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16">
    <h2 class="text-4xl md:text-6xl lg:text-7xl xl:text-8xl">
      Connect
    </h2>
    <img class="object-cover py-4 max-h-72" src="../../img/connect.jpg" alt="Dominic walking - only his legs are visible. Cropped, dark gray jeans. Brown leather brogue shoes.">
    <div class="grid grid-cols-1 sm:grid-cols-2" data-scroll data-scroll-speed="0.5">
      <p class="hidden text-sm xl:text-3xl md:text-xl lg:text-2xl sm:block">I am available for freelance projects and <span class="whitespace-nowrap">full-time</span> employment.</p>
      <div class="flex flex-col space-y-2">
        <a class="ml-auto text-4xl transition duration-300 lg:text-5xl hover:opacity-60" href="mailto:contactdomferris@gmail.com">
          Email
        </a>
    
        <a class="ml-auto text-4xl transition duration-300 lg:text-5xl hover:opacity-60" href="https://github.com/domferris" target="_blank">
          GitHub
        </a>
        
        <a class="ml-auto text-4xl transition duration-300 lg:text-5xl hover:opacity-60" href="https://www.linkedin.com/in/domferris/" target="_blank">
          LinkedIn
        </a>
    
        <a class="ml-auto text-4xl transition duration-300 lg:text-5xl hover:opacity-60" href="https://www.instagram.com/domferris/" target="_blank">
          Instagram
        </a>
    
        <a class="ml-auto text-4xl transition duration-300 lg:text-5xl hover:opacity-60" href="https://twitter.com/domferris" target="_blank">
          Twitter
        </a>
    
        <a class="ml-auto text-4xl transition duration-300 lg:text-5xl hover:opacity-60" href="/blog">
          Blog
        </a>
      </div>
    </div>
    <div class="flex items-end justify-center flex-grow">
      <p class="text-base text-center lg:text-lg">Thanks for visiting<br>â˜º</p>
    </div>
  </div>
</section>

</body>
</html>
