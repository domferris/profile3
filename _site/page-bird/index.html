<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>
    Dominic Ferris - Page Bird

  </title>

  <!-- Fontawesome Stylesheet -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">

  <!-- Styles bundle -->
  <link rel="stylesheet" href="/assets/main.bundle.css">

  <!-- Locomotive Scroll polyfills -->
  <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.6.0/polyfill.min.js"
    crossorigin="anonymous"></script>
  <script nomodule
    src="https://polyfill.io/v3/polyfill.min.js?features=Object.assign%2CElement.prototype.append%2CNodeList.prototype.forEach%2CCustomEvent%2Csmoothscroll"
    crossorigin="anonymous"></script>

  <!-- JavaScript bundle -->
  <script src="/assets/main.bundle.js" defer></script>
</head>
<body class="bg-project text-true-gray-800" data-scroll-container>
  
<section class="max-w-md min-h-screen p-4 sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16">
  <a class="items-center inline-block text-base transition duration-300 md:text-xl lg:text-2xl hover:opacity-60" href="/#projects">
    &lt; View other projects
  </a>
</section>
</body>
</html>
