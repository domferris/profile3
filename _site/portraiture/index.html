<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>
    Dominic Ferris - Portraiture

  </title>

  <!-- Fontawesome Stylesheet -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">

  <!-- Styles bundle -->
  <link rel="stylesheet" href="/assets/main.bundle.css">

  <!-- Locomotive Scroll polyfills -->
  <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.6.0/polyfill.min.js"
    crossorigin="anonymous"></script>
  <script nomodule
    src="https://polyfill.io/v3/polyfill.min.js?features=Object.assign%2CElement.prototype.append%2CNodeList.prototype.forEach%2CCustomEvent%2Csmoothscroll"
    crossorigin="anonymous"></script>

  <!-- JavaScript bundle -->
  <script src="/assets/main.bundle.js" defer></script>
</head>
<body class="bg-project text-true-gray-800" data-scroll-container>
  
<div data-scroll-section>
  <section class="max-w-md min-h-screen p-4 mx-auto sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16">
    <h2 class="text-4xl md:mb-2 xl:mb-5 md:text-6xl lg:text-7xl xl:text-8xl">
      Portraiture
    </h2>
  
    <h4 class="text-lg mt-1.5 md:mt-3.5 mb-3 md:mb-4 xl:mb-5 whitespace-nowrap md:text-2xl lg:text-3xl xl:text-4xl">
      Photo series
    </h4>

    <p class="lg:text-lg md:mb-7">
      After years of experimenting with many forms of photography, portrait, street, and documentary photography is where I found my passion. My approach to portraiture is an almagamation of those styles. Though I no longer advertise myself as a photographer, I still enjoy this creative expression and still shoot to this day.
    </p>

    <div class="flex flex-col items-center mt-4 space-y-8 md:my-7">
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-01.jpg" alt="Portrait of Michael">
        <p class="text-sm">Michael O</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-02.jpg" alt="Behind-the scenes potrait during Stela Cole's Lucky Day music video shoot">
        <p class="text-sm">Stela Cole Glam Chair</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-03.jpg" alt="Ryan drinking coffee at S&W Country Diner">
        <p class="text-sm">S&W Ryan Pollie</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-04.jpg" alt="Nadeen at Pilsen Community Books">
        <p class="text-sm">Pilsen Community Books</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-05.jpg" alt="Emmitt laying on a mattress recording his next album">
        <p class="text-sm">Recording Emmitt James</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-06.jpg" alt="Portrait of Chloe">
        <p class="text-sm">Chloe in Highland Park</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-07.jpg" alt="Alex outside in Beverly Hills">
        <p class="text-sm">Bohs on Break</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-08.jpg" alt="Christine at Leo Carrillo State Beach">
        <p class="text-sm">Memorial Day Malibu</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-09.jpg" alt="Yoskay Yamamoto in his studio">
        <p class="text-sm">Yoskay's studio</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-10.jpg" alt="Rafael at the Farmer's Market">
        <p class="text-sm">Farmer's Market Pa-el</p>
      </div>
      <div class="space-y-2">
        <img class="max-h-92" src="../../img/photography/photography-11.jpg" alt="Brendan sitting outside Fuller House">
        <p class="text-sm">Fuller BLS</p>
      </div>
    </div>

    <p class="lg:text-lg my-7">
      View more of my photography on <a class="text-transparent transition duration-300 bg-clip-text bg-gradient-to-r from-purple-700 via-red-500 to-yellow-700 hover:opacity-70" href="https://www.instagram.com/domferris/" target="_blank">Instagram</a>.
    </p>
    
    <div class="flex flex-col mt-10 md:mt-12 lg:mt-16">
      <div class="flex">    
        <h2 class="pr-4 ml-auto text-4xl md:text-6xl lg:text-7xl">
          Related
        </h2>

        <span class="w-full my-auto md:h-1.5 h-1 lg:h-2 bg-true-gray-800"></span>
      </div>
  
      <a class="items-end inline my-4 mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 md:my-5 lg:my-6 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/marches">
        Marches<span class="hidden text-base lg:text-lg sm:inline"> Photo series</span>
      </a>
      
      <a class="inline-block mr-auto transition duration-300 md:text-xl lg:text-2xl hover:opacity-60" href="/#projects">
        &lt; View other projects
      </a>
    </div>
  </section>
</div>
</body>
</html>
