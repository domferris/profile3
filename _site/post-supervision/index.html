<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>
    Dominic Ferris - Post Supervision

  </title>

  <!-- Fontawesome Stylesheet -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">

  <!-- Styles bundle -->
  <link rel="stylesheet" href="/assets/main.bundle.css">

  <!-- Locomotive Scroll polyfills -->
  <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.6.0/polyfill.min.js"
    crossorigin="anonymous"></script>
  <script nomodule
    src="https://polyfill.io/v3/polyfill.min.js?features=Object.assign%2CElement.prototype.append%2CNodeList.prototype.forEach%2CCustomEvent%2Csmoothscroll"
    crossorigin="anonymous"></script>

  <!-- JavaScript bundle -->
  <script src="/assets/main.bundle.js" defer></script>
</head>
<body class="text-true-gray-800" data-scroll-container>
  
<div class="bg-true-gray-300" data-scroll-section>
  <section class="max-w-md min-h-screen p-4 mx-auto sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16">
    <h2 class="text-4xl md:mb-2 xl:mb-5 md:text-6xl lg:text-7xl xl:text-8xl">
      Post Supervision
    </h2>
  
    <div class="flex flex-wrap items-center justify-between mb-2 md:mb-4">
      <h4 class="text-lg whitespace-nowrap md:text-2xl lg:text-3xl xl:text-4xl">
        Ad collection
      </h4>
  
      <a class="inline-block p-1 text-2xl transition duration-300 lg:text-4xl md:text-3xl hover:opacity-60 xl:5xl" href="https://github.com/spicygreenbook/greenbook-app" target="_blank">
        <i class="fab fa-vimeo-v"></i>
      </a>
    </div>
  
    <img class="mb-3 md:mb-6" src="../../img/projects/sgb-home.png" alt="Spicy Green Book homepage banner">
    
    <div class="space-y-4 md-space-y-8 lg:text-lg">
      <p>
        While working at a ViacomCBS ad agency, I supervised post production on numerous ad campaigns for brands such as Google Pixel, McDonalds, Folgers, Target, Pampers, and many more.
      </p>
  
      <p>
        Because this collection of projects contains videos I do not own, please reach out to receive the password in order to acces this private collection on Vimeo.
      </p>
    </div>

    <div class="grid grid-cols-1 gap-4 my-4 md:grid-cols-2 md:my-7">
      <img src="../../img/projects/sgb-directory.png" alt="">
      
      <img src="../../img/projects/sgb-directory.png" alt="">
      
      <img src="../../img/projects/sgb-directory.png" alt="">
      
      <img src="../../img/projects/sgb-directory.png" alt="">
      
      <img class="md:col-span-2" src="../../img/projects/sgb-directory.png" alt="">
    </div>
    
    <div class="flex flex-col mt-10 md:mt-12 lg:mt-16">
      <div class="flex">    
        <h2 class="pr-4 ml-auto text-4xl md:text-6xl lg:text-7xl">
          Related
        </h2>

        <span class="w-full my-auto md:h-1.5 h-1 lg:h-2 bg-true-gray-800"></span>
      </div>
  
      <a class="items-end inline my-4 mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 md:my-5 lg:my-6 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/power-through">
        Power Through<span class="hidden text-base lg:text-lg sm:inline"> Spec commercial</span>
      </a>
      
      <a class="inline-block mr-auto transition duration-300 md:text-xl lg:text-2xl hover:opacity-60" href="/#projects">
        &lt; View other projects
      </a>
    </div>
  </section>
</div>
</body>
</html>
