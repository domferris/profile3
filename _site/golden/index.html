<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <title>
    Dominic Ferris - Golden

  </title>

  <!-- Fontawesome Stylesheet -->
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">

  <!-- Styles bundle -->
  <link rel="stylesheet" href="/assets/main.bundle.css">

  <!-- Locomotive Scroll polyfills -->
  <script nomodule src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.6.0/polyfill.min.js"
    crossorigin="anonymous"></script>
  <script nomodule
    src="https://polyfill.io/v3/polyfill.min.js?features=Object.assign%2CElement.prototype.append%2CNodeList.prototype.forEach%2CCustomEvent%2Csmoothscroll"
    crossorigin="anonymous"></script>

  <!-- JavaScript bundle -->
  <script src="/assets/main.bundle.js" defer></script>
</head>
<body class="bg-true-gray-300 text-true-gray-800" data-scroll-container>
  
<div data-scroll-section>
  <section class="max-w-md min-h-screen p-4 mx-auto sm:max-w-xl md:p-8 md:max-w-4xl lg:max-w-7xl lg:p-16">
    <h2 class="text-4xl md:mb-2 xl:mb-5 md:text-6xl lg:text-7xl xl:text-8xl">
      Golden
    </h2>
  
    <h4 class="text-lg mt-1.5 md:mt-3.5 mb-3 md:mb-4 xl:mb-5 whitespace-nowrap md:text-2xl lg:text-3xl xl:text-4xl">
      Music video
    </h4>
    
    <div class="relative mb-3 aspect-w-16 aspect-h-9 md:mb-6">
      <iframe class="z-10 w-full" title="vimeo-player" src="https://player.vimeo.com/video/224732536?color=D4D4D4&title=0&byline=0&portrait=0" frameborder="0" allowfullscreen></iframe>
      <div class="absolute top-0 left-0 w-full h-full bg-black"></div>
    </div>

    <p class="lg:text-lg md:mb-7">
      The official music video for the song "Golden" by musician Nanna B shot on location at the Vasquez Rocks. The track features Hodgy and is produced by Anderson .Paak.
    </p>

    <div class="grid grid-cols-1 gap-4 my-4 md:grid-cols-2 md:my-7">
      <img src="../../img/projects/golden-01.png" alt="Profile of Nanna B">
      <img src="../../img/projects/golden-02.png" alt="Hodgy pulling on golden fabric leading to Nanna B">
      <img src="../../img/projects/golden-03.png" alt="Close-up of Nanna B looking at Hodgy">
      <img src="../../img/projects/golden-04.png" alt="Nanna B and Hodgy atop rocks">
    </div>
    
    <div class="flex flex-col mt-10 md:mt-12 lg:mt-16">
      <div class="flex">
        <h2 class="pr-4 ml-auto text-4xl md:text-6xl lg:text-7xl">
          Related
        </h2>

        <span class="w-full my-auto md:h-1.5 h-1 lg:h-2 bg-true-gray-800"></span>
      </div>
  
      <div class="flex flex-col my-4 md:space-y-1 md:my-5 lg:space-y-2 lg:my-6">
        <a class="items-end inline mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/directing">
          Directing<span class="hidden text-base lg:text-lg sm:inline"> Video reel</span>
        </a>

        <a class="items-end inline mr-auto text-2xl transition duration-300 md:text-3xl sm:mr-0 sm:flex sm:justify-between lg:text-4xl hover:opacity-60" href="/corduroy-velvet">
          Corduroy Velvet<span class="hidden text-base lg:text-lg sm:inline"> Short film</span>
        </a>
      </div>
      
      <a class="inline-block mr-auto transition duration-300 md:text-xl lg:text-2xl hover:opacity-60" href="/#projects">
        &lt; View other projects
      </a>
    </div>
  </section>
</div>
</body>
</html>
